src/ng/filter/orderBy.js

```diff
@@ -177,7 +177,7 @@ orderByFilter.$inject = ['$parse'];
 function orderByFilter($parse) {
   return function(array, sortPredicate, reverseOrder) {
 
-    if (!(isArrayLike(array))) return array;
+    if (!(isArrayLike(array))) throw minErr('orderBy')('notarray', 'Expected array but received: {0}', array);
```

Comment:
@gkalpak  I guess something like 

``` javascript
if(!array) return array;

if (!(isArrayLike(array))) throw minErr('orderBy')('notarray', 'Expected array but received: {0}', array);
```

would be easier to read than the example.

This is just an idea,of course :) what do you think about that?